virt/virtual_machines/virtual_disks/virt-vm-disk-resizing.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="virt-vm-disk-resizing"]
+= Virtual machine disk resizing
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-vm-disk-resizing
+
+toc::[]
+
+You can enlarge the size of a virtual machine's (VM) disk to provide a greater storage capacity.
+
+However, you cannot reduce the size of a VM disk.
+
+include::modules/virt-enlarging-vm-disk.adoc[leveloffset=+1]
+
+[id="additional-resources_virt-vm-disk-resizing"]
+== Additional resources
+
+* Microsoft provides additional information for how to link:https://docs.microsoft.com/en-us/windows-server/storage/disk-management/extend-a-basic-volume[extend a basic volume].
```

Comment:
Instead of explicitly calling out Microsoft, I'd suggest just keeping the link (no text before the link) and changing the link text to [Extending a basic volume in Windows]